release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -638,6 +638,11 @@ The Insights Operator now gathers more data to detect the following scenarios, w
 [id="ocp-release-notes-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-18-installation-cluster-api-provider-ibmcloud_{context}"]
+==== New version of `cluster-api-provider-ibmcloud`
+
+The installation program now uses a newer version of the `cluster-api-provider-ibmcloud` provider that includes Transit Gateway fixes. Because of the cost of Transit Gateways in {ibm-cloud-title}, you can now use the {product-title} to create a Transit Gateway when creating an {product-title} cluster. For more information, see (link:https://issues.redhat.com/browse/OCPBUGS-37588[*OCPBUGS-37588*]) and (link:https://issues.redhat.com/browse/OCPBUGS-41938[*OCPBUGS-41938*]).
```

Comment:
There is an attribute for the correct name of this
```suggestion
==== New version of the {cap-ibm-short}

The installation program now uses a newer version of the {cap-ibm-short} that includes Transit Gateway fixes. Because of the cost of Transit Gateways in {ibm-cloud-title}, you can now use the {product-title} to create a Transit Gateway when creating an {product-title} cluster. For more information, see (link:https://issues.redhat.com/browse/OCPBUGS-37588[*OCPBUGS-37588*]) and (link:https://issues.redhat.com/browse/OCPBUGS-41938[*OCPBUGS-41938*]).
```